src/info/manifest.c

```diff
@@ -1542,6 +1543,25 @@ manifestBuildComplete(
             lstSort(this->pub.dbList, sortOrderAsc);
         }
 
+        // Save annotations
+        if (annotation != NULL)
+        {
+            this->pub.data.annotation = varNewKv(kvNew());
+
+            KeyValue *const manifestAnnotationKv = varKv(this->pub.data.annotation);
+            const VariantList *const annotationKeyList = kvKeyList(annotation);
+
+            for (unsigned int keyIdx = 0; keyIdx < varLstSize(annotationKeyList); keyIdx++)
+            {
+                const Variant *const key = varLstGet(annotationKeyList, keyIdx);
+                const Variant *const value = kvGet(annotation, key);
+
+                // Skip empty values
+                if (!strEmpty(varStr(value)))
+                    kvPut(manifestAnnotationKv, key, value);
```

Comment:
Not entirely satisfied with this. If the user specifies a lot of empty values then we just store an empty KV? Shouldn't we error if the initial values are empty?

Reply:
I added some clean-up steps in 4db7d9a. 
With that clean-up, I don't think it is necessary to add the warning in the backup output (in case all provided annotations values are empty) given all empty values are skipped and the manifest/backup.info field isn't created anyway.